### DISCUSSION TITLE
Could I use other models besides GPT?

### DISCUSSION THREAD

**Discussion by thisisfangsheng** [microsoft/autogen#146] [Q&A]
As title. I wonder if I could use other models instead of GPT. Thanks.

**Comment by sonichi** [comment]
Yes. Please see #45 for an example. Also check https://microsoft.github.io/autogen/blog/2023/07/14/Local-LLMs

**Reply by lzu-tangym** [reply]
Could I use GLM or MOSS of CN besides GPT？

**Reply by sonichi** [reply]
@LeoLjl
